packages/material/modules/widgets/value/MaterialAutocomplete.jsx

```diff
@@ -73,6 +73,10 @@ export default (props) => {
     minWidth: minWidth
   };
   const placeholder = !readonly ? aPlaceholder : "";
+
+  // For accessibility, always give the input field an aria-label
+  const ariaLabel = config.settings.fieldPlaceholder;
```

Comment:
Could you please change this to `const ariaLabel = placeholder  || config.settings.fieldPlaceholder`  ?

`MaterialAutocomplete` can be used to select field in LHS (then `aPlaceholder` is null and it's correct to use `fieldPlaceholder`) but also can be used in RHS (see "Autocomplete" field in the example app)

Reply:
And same for `MuiAutocomplete ` please

Note: `Mui*` are for v5, `Material*` are for obsolete v4.